Modules/posixmodule.c

```diff
@@ -10073,9 +10073,29 @@ os__getdiskusage_impl(PyObject *module, path_t *path)
     Py_BEGIN_ALLOW_THREADS
     retval = GetDiskFreeSpaceExW(path->wide, &_, &total, &free);
     Py_END_ALLOW_THREADS
-    if (retval == 0)
+    if (retval == 0) {
+        if (GetLastError() == ERROR_DIRECTORY) {
+            wchar_t *dir_path = NULL;
+
+            dir_path = PyMem_New(wchar_t, path->length + 1);
+            if (dir_path == NULL)
+                return PyErr_NoMemory();
```

Comment:
PEP 7 now requires braces even for single line blocks (which I'm sad about, but it's our style guide now)

Reply:
Cool - I was just following the convention I saw in the file, but I'm happy to add the braces.